.gitignore

```diff
@@ -229,34 +115,13 @@ DNN [Pp]latform/[Pp]roviders/*/[Bb]in/*
 *.nl-NL.resx
 
 # but do track translations in the Install folder
-![Ww]ebsite/[Ii]nstall/[Aa]pp_[Ll]ocal[Rr]esources/*.de-DE.resx
-![Ww]ebsite/[Ii]nstall/[Aa]pp_[Ll]ocal[Rr]esources/*.es-ES.resx
-![Ww]ebsite/[Ii]nstall/[Aa]pp_[Ll]ocal[Rr]esources/*.fr-FR.resx
-![Ww]ebsite/[Ii]nstall/[Aa]pp_[Ll]ocal[Rr]esources/*.it-IT.resx
-![Ww]ebsite/[Ii]nstall/[Aa]pp_[Ll]ocal[Rr]esources/*.nl-NL.resx
-
-*.zip.manifest
-
-############
-## Windows
-############
+!DNN Platform/[Ww]ebsite/[Ii]nstall/[Aa]pp_[Ll]ocal[Rr]esources/*.de-DE.resx
+!DNN Platform/[Ww]ebsite/[Ii]nstall/[Aa]pp_[Ll]ocal[Rr]esources/*.es-ES.resx
+!DNN Platform/[Ww]ebsite/[Ii]nstall/[Aa]pp_[Ll]ocal[Rr]esources/*.fr-FR.resx
+!DNN Platform/[Ww]ebsite/[Ii]nstall/[Aa]pp_[Ll]ocal[Rr]esources/*.it-IT.resx
+!DNN Platform/[Ww]ebsite/[Ii]nstall/[Aa]pp_[Ll]ocal[Rr]esources/*.nl-NL.resx
 
-# Windows image file caches
-Thumbs.db
+# Add fips back
+!DNN Platform/[Ww]ebsite/App_Data/FipsCompilanceAssemblies/Lucene.Net.dll
 
-# Folder config file
-Desktop.ini
-Website/Install/Temp/
-Website/Providers/HtmlEditorProviders/
-Website/Portals/_default/CK*.xml
-Website/Portals/_default/Install/CK*.xml
-Website/Portals/_default/Skins/Xcillion/
-Website/Portals/_default/Containers/Xcillion/
-DNN Platform/Dnn.AuthServices.Jwt/Package/
-Website/Portals/_default/Install/Dnn.CKEditorDefaultSettings.xml
-Website/Portals/_default/Install/Dnn.CKToolbarButtons.xml
-Website/Portals/_default/Install/Dnn.CKToolbarSets.xml
-Website/DesktopModules/Admin/SiteExportImport/App_LocalResources/ExportImport.resx
-/TestResult.xml
-/Website/DesktopModules/Admin/Dnn.EditBar
-/Website/DesktopModules/Admin/Dnn.PersonaBar
+D:\DNN\Dnn.AdminExperience\Extensions\Content\Dnn.PersonaBar.Extensions\admin\personaBar\Dnn.Roles\scripts\bundles\rw-widgets.svg
```

Comment:
This absolute path shouldn't be in the `.gitignore`, right?

Reply:
True ... I'll repair

Reply:
But if it's OK, I'll do that after merge as otherwise I have to do a lot of resolving locally given I'm out of sync.

Reply:
I went ahead and pushed that change